app/views/users/show.html.erb

```diff
@@ -1 +1,32 @@
-<h1>User details</h1>
\ No newline at end of file
+<div class="user-item">
+  <div class="user-image">
+    <%= image_tag @user.photo, alt: @user.name, width: '100' %>
+  </div>
+  <div class="user-info">
+    <h2><%= @user.name %></h2>
+    <div class="user-posts">
+      <p>Number of Posts: <%= @user.posts_counter %></p>
+    </div>
+  </div>
+</div>
+<div class="bio">
+  <h2>Bio</h2>
+  <p><%= @user.bio %></p>
+</div>
+
+<div class="posts">
+  <% @user.posts.order(created_at: :desc).limit(3).each do |post| %>
```

Comment:
- [ ] While you did a fantastic job as a project requirement you should use the method #recent_post method to render the user's 3 most recent post 👍